lib/ReactViews/Analytics/PointParameterEditor.jsx

```diff
@@ -50,31 +112,6 @@ const PointParameterEditor = createReactClass({
     },
 });
 
-/**
- * Triggered when user types value directly into field.
- * @param {String} e Text that user has entered manually.
- * @param {FunctionParameter} parameter Parameter to set value on.
- */
-PointParameterEditor.setValueFromText = function(e, parameter) {
-    const coordinates = e.target.value.split(',');
-    if (coordinates.length >= 2) {
-        parameter.value = Cartographic.fromDegrees(parseFloat(coordinates[0]), parseFloat(coordinates[1]));
-    }
-};
-
-/**
- * Given a value, return it in human readable form for display.
- * @param {Object} value Native format of parameter value.
- * @return {String} String for display
- */
-PointParameterEditor.getDisplayValue = function(value) {
```

Comment:
This function is part of the public API and used in Geoglam:
https://github.com/TerriaJS/geoglam-nm/blob/c945f4fbf6181121c83744018b03c44e5b657a58/lib/Views/GeoJsonParameterEditor.jsx#L106

I don't know if `setValueFromText` is used, but it's part of the public API too.